docs/get-started/performance-reports.md

```diff
@@ -0,0 +1,119 @@
+# Performance Reports
+
+Our goal is to make a highly-performant, feature-rich and easy to setup and maintain blcokchain client software.
+
+All tests were done using the [Polygon Edge Loadbot](../additional-features/stress-testing.md).
+
+Every performance report you will find on this page is properly dated, environment clearly described and the testing method clearly explained.
+
+## January 21st 2022
+
+### Summary
+
+This test was done after the TxPool refactor which significantly improved performance (released in [v0.2.0](https://github.com/0xPolygon/polygon-edge/releases/v0.2.0)).
+
+The goal was to setup a large network consisting of 30 actively participating validators in order to properly stress test the
+consensus and TxPool transaction gossiping as all transactions were sent to a single node's JSON-RPC.
+
+Our aim was not to strive to reach a maximum possible TPS, as the network size negatively impacts the performance,
+and since block gas limit & block time are set to sane values that don't consume much system resources, and would allow this to run on commodity hardware.
+
+### Results
+
+| Metric | Value |
+| ------ | ----- |
+| Transactions per second | 344 |
+| Transactions failed | 0 |
+| Transactions succeeded | 10000 |
+| Total run time | 30s |
+
+### Environment
+
+<details>
+  <summary>Host Configuration</summary>
+  <div>
+    <div>
+        <table>
+            <tr>
+                <td>Cloud provider</td>
+                <td>AWS</td>
+            </tr>
+            <tr>
+                <td>Instance size</td>
+                <td>t2.xlarge</td>
+            </tr>
+            <tr>
+                <td>Networking</td>
+                <td>private subnet</td>
+            </tr>
+            <tr>
+                <td>Operating system</td>
+                <td>Linux Ubuntu 20.04 LTS - Focal Fossa</td>
+            </tr>
+            <tr>
+                <td>File descriptor limit</td>
+                <td>65535</td>
+            </tr>
+            <tr>
+                <td>Max user processes</td>
+                <td>65535</td>
+            </tr>
+        </table>
+    </div>
+    <br/>
+  </div>
+</details>
+
+<details>
+  <summary>Blockchain Configuration</summary>
+  <div>
+    <div>
+        <table>
+            <tr>
+                <td>Validator nodes</td>
+                <td>30</td>
+            </tr>
+            <tr>
+                <td>Non-validator nodes</td>
+                <td>0</td>
+            </tr>
+            <tr>
+                <td>Consensus</td>
+                <td>IBFT PoA</td>
+            </tr>
+            <tr>
+                <td>Block time</td>
+                <td>20000ms</td>
```

Comment:
There is a typo here, it should be `2000ms` (`2s`)

Reply:
Thank you @zivkovicmilos https://github.com/0xPolygon/polygon-edge-docs/pull/63/commits/cff7204b31a447d37cc6c7adf9aa9f958e182c4e